### ISSUE TITLE
Email address field on Customer Details screen is missing a label

### ISSUE THREAD

**Issue by robincornett** [easydigitaldownloads/easy-digital-downloads#7725]
### Expected behavior
Form elements should have labels. For assistive technology users, a placeholder is an insufficient replacement for a `<label>` element.

### Actual behavior
The email address field on the Customer Details screen does not have any kind of label.

### Steps to reproduce the behavior
1) On the Customer Details screen, inspect the field for adding an email address to the customer's profile. There is no label.

### Information (if a specific version is affected):
PHP Version: 7.3.5
EDD Version (or branch): master, release/3.0
WordPress Version: 5.4

Any other relevant information:
Adding a label element which is visually hidden will improve the accessibility of this input field.

Related: the "Primary" `span` (class `edd-chip`) has insufficient color contrast (1.91) and is very small text.

**Comment by spencerfinnell** [comment]
@robincornett For the chip I think we could set the background color to the core text color `#444`, which provides plenty of contrast. I think the font size could be bumped as well like you mentioned, and removing the border would give us a few extra pixels to work with.

**Comment by spencerfinnell** [comment]
@LisaCee I think this is a good one for you to tackle. Take a look at https://make.wordpress.org/accessibility/handbook/markup/the-css-class-screen-reader-text/. We still want just the placeholder to be visible, but we need an actual `label` for assistive technology like screen readers.

The input @robincornett is referring to is located here:

https://github.com/easydigitaldownloads/easy-digital-downloads/blob/8781ddd686f13b526bb08a59ddaeec40b7a796e4/includes/admin/customers/customers.php#L416

Or Downloads > Customers > Edit Customer > Customer Emails in the dashboard.

**Comment by spencerfinnell** [comment]
Separate issue for the Chip component accessibility here: https://github.com/easydigitaldownloads/easy-digital-downloads/issues/7731

**Comment by robincornett** [comment]
The label could maybe be fixed for 2.9.x, actually, and then I guess merged into 3.0 later, as it's an issue in both versions (my mistake on initial report, sorry).

**Comment by spencerfinnell** [comment]
@robincornett In an effort to keep `2.9.x` releases small we are only including more critical bug fixes  and putting improvements into `release/3.0`.

**Comment by LisaCee** [comment]
PR #7734
